contracts/beacon/Voucher.sol

```diff
@@ -36,6 +37,12 @@ contract Voucher is OwnableUpgradeable, IVoucher {
         mapping(bytes32 dealId => uint256) _sponsoredAmounts;
         // Save refunded tasks to disable replay attacks.
         mapping(bytes32 taskId => bool) _refundedTasks;
+        address _owner;
```

Comment:
Minor, I suggest to keep in storage a consistant order with args from initialize
```
        address voucherOwner,
        address voucherHub,
        uint256 expiration,
        uint256 voucherTypeId
```
(so owner is moved to top)

Reply:
Done [here](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/22/commits/97ec801106d4742d09af78e4b9d431dc00b46f9c)